engine/battle/ai/scoring.asm

```diff
@@ -443,6 +459,16 @@ AI_Smart_LeechHit:
 	ret
 
 AI_Smart_LockOn:
+; Greatly encourage (-2) moves with 70% accuracy or less if the player is already locked on.
+; Dismiss (-10) Lock On move if the player is already locked on.
+; Discourage (+1) Lock On move if the enemy hp is : 
+;   - ] 0%; 25%].
+;   - ]25%; 50%] and the enemy is strictly slower.
+;   - ]50%;100%] and (player evasion level is negative or enemy accuracy level is positive)
```

Comment:
Weird format

Reply:
Since i'm mostly documenting for speedrunning purposes, i like to know whether a bound is included or not. But i have to admit i'm still working on routines that mix a lot of different behaviours. Trying to make them more readable.

Reply:
Mathematical notation is (0%, 25%]

Reply:
Anglosaxon notation is yours. "Southern" European notation (In France, Spain, Italy, Germany, etc.) is the one i used. I'll change this if it's a real understanding issue.